pkg/kf/commands/install/kf/kf.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kf
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path"
+
+	. "github.com/google/kf/pkg/kf/commands/install/util"
+)
+
+const (
+	// KnativeBuildYAML holds the knative build yaml release URL
+	KnativeBuildYAML = "https://github.com/knative/build/releases/download/v0.7.0/build.yaml"
+	// KfNightlyBuildYAML holds the kf nightly build release URL
+	KfNightlyBuildYAML = "https://storage.googleapis.com/artifacts.kf-releases.appspot.com/nightly-builds/releases/release-latest.yaml"
```

Comment:
Should we display which version this command installs, especially kf version?

Reply:
So I like this, but it might be tricky until we start cutting actual version. When we have 0.x versions, we can pin them and report what we've pinned.

Alternatively, we can add meta to the nightly releases and read from that. Either way I think I'll kick this to a new issue https://github.com/google/kf/issues/456 so we can discuss it further.